### ISSUE TITLE
GetScheduleAsync exception

### ISSUE THREAD

**Issue by JaoHundred** [Ervie/jikan.net#42]
Hi Ervie, I think I found some sort of bug from GetScheduleAsync method, this was what I found.

```
{System.InvalidOperationException: The JSON property name for 'JikanDotNet.DbPaginatedJikanResponse`1[System.Collections.Generic.ICollection`1[JikanDotNet.Anime]].Pagination' collides with another property.
  at System.Text.Json.ThrowHelper.ThrowInvalidOperationException_SerializerPropertyNameConflict (System.Text.Json.JsonClassInfo jsonClassInfo, System.Text.Json.JsonPropertyInfo jsonPropertyInfo) [0x0001b] in <815ff95f4d6a463fa6a3be490e0514aa>:0 
  at System.Text.Json.JsonClassInfo..ctor (System.Type type, System.Text.Json.JsonSerializerOptions options) [0x00133] in <815ff95f4d6a463fa6a3be490e0514aa>:0 
  at System.Text.Json.JsonSerializerOptions.GetOrAddClass (System.Type classType) [0x00017] in <815ff95f4d6a463fa6a3be490e0514aa>:0 
  at System.Text.Json.ReadStackFrame.Initialize (System.Type type, System.Text.Json.JsonSerializerOptions options) [0x00000] in <815ff95f4d6a463fa6a3be490e0514aa>:0 
  at System.Text.Json.JsonSerializer.ReadCore (System.Type returnType, System.Text.Json.JsonSerializerOptions options, System.Text.Json.Utf8JsonReader& reader) [0x00008] in <815ff95f4d6a463fa6a3be490e0514aa>:0 
  at System.Text.Json.JsonSerializer.Deserialize (System.String json, System.Type returnType, System.Text.Json.JsonSerializerOptions options) [0x0009d] in <815ff95f4d6a463fa6a3be490e0514aa>:0 
  at System.Text.Json.JsonSerializer.Deserialize[TValue] (System.String json, System.Text.Json.JsonSerializerOptions options) [0x00000] in <815ff95f4d6a463fa6a3be490e0514aa>:0 
  at JikanDotNet.Jikan.ExecuteGetRequestAsync[T] (System.Collections.Generic.ICollection`1[T] routeSections) [0x0012a] in <210aaa08ca6e48cc9535ceee605c1cbc>:0 
```

**Comment by Ervie** [comment]
I tried it a few times myself, but it passes every time. Were you calling it with some specific parameter? Could be just some issues on API side (like returning pagination object twice), but I'm unsure.

**Comment by JaoHundred** [comment]
I was using the ScheduledDay parameter, didn't tried the parameterless nor the int one. Later I will try to investigate more.

**Comment by JaoHundred** [comment]
I Tried a few things: 

- Parameterless, int, ScheduledDay
- Calling the method from main thread
- Checking if the method was being called twice

Didn't found the issue source for now, this only happened from my experience when I used the DbPaginatedJikanResponse.

**Comment by Ervie** [comment]
Ok, please let me know if you are able to reproduce the issue I tried few other things but to no avail too.

**Comment by JaoHundred** [comment]
I made a console app to test this and used .NET Core 3.1 framework, the same exception message appears to me for every request which returns DbPaginatedJikanResponse, I can confirm now it only happens for this specific type. Maybe could this have some incompatibility with the operating system language and the underlying Json text?

**Comment by Ervie** [comment]
My take (not confirmed) is that it might be affecting older supported .net versions, meaning .netstandard2.0 and .netcore3.1. Honestly I keep my code and tests in the newest version (currently net6.0), so it might use newer syntax that doesn't translate well to older versions. I'll try to verify that later.

**Comment by JaoHundred** [comment]
I see, that does make sense, I'm using .NET Standard 2.1 for my app. I only have the tools for standard 2.1 and core 3.1.

**Comment by Ervie** [comment]
I can confirm that this is happening for version core 3.1 and possible .netstandard2.0. I'll try to find to proper remedy next weekend.

**Comment by JaoHundred** [comment]
Thanks again for the help Ervie.

**Comment by Ervie** [comment]
Hello,

I pushed the package with version 2.4.0 - it should use standardized model compatible with versions 3.1 and older. The changes are described in the readme and changelog.

**Comment by JaoHundred** [comment]
It works now, thanks Ervie.

**Comment by Ervie** [comment]
Solved in version 2.4.0
